core/serialization/workspaces.js

```diff
@@ -58,23 +59,33 @@ exports.save = save;
  * @param {!Object<string, *>} state The state of the workspace to deserialize
  *     into the workspace.
  * @param {!Workspace} workspace The workspace to add the new state to.
+ * @param {{recordUndo: (boolean|undefined)}=} param1
+ *     recordUndo: If true, events triggered by this function will be undo-able
+ *       by the user. False by default.
```

Comment:
I'm going to keep this because I think it makes the docs more readable when you have multiple optional parameters. Eg:
```
 * @param {{addCoordinates: (boolean|undefined), addInputBlocks:
 *     (boolean|undefined), addNextBlocks: (boolean|undefined)}=} param1
 *     addCoordinates: If true, the coordinates of the block are added to the
 *       serialized state. False by default.
 *     addinputBlocks: If true, children of the block which are connected to
 *       inputs will be serialized. True by default.
 *     addNextBlocks: If true, children of the block which are connected to the
 *       block's next connection (if it exists) will be serialized.
 *       True by default.
```
vs:
```
 * @param {{addCoordinates: (boolean|undefined), addInputBlocks:
 *     (boolean|undefined), addNextBlocks: (boolean|undefined)}=} param1
 *     addCoordinates: If true, the coordinates of the block are added to the
 *     serialized state. False by default.
 *     addinputBlocks: If true, children of the block which are connected to
 *     inputs will be serialized. True by default.
 *     addNextBlocks: If true, children of the block which are connected to the
 *     block's next connection (if it exists) will be serialized.
 *     True by default.
```
The [styleguide](https://google.github.io/styleguide/jsguide.html#features-objects-destructuring) doesn't seem to have an opinion.